kube/ims-resolver/ims-resolver-deploy.yml

```diff
@@ -39,7 +39,7 @@ spec:
     spec:
       containers:
         - name: ims-resolver
-          image: quay.io/ukhomeofficedigital/ims-resolver:6802b5b1f913ec1b46ed7e5bd2f40958448ab1c7
+          image: quay.io/ukhomeofficedigital/ims-resolver:a12778b49fa70f36b733768909ff7ddea62e9efb
```

Comment:
this matches the current commit hash (`a12778b49fa70f36b733768909ff7ddea62e9efb`) following the merge of PR 40 (https://github.com/UKHomeOffice/ims-resolver/pull/40) 👍